src/components/common/header/Header.tsx

```
@@ -16,20 +16,22 @@ import { formatImgPath } from '../../../util/formatImgPath';
 import bell from '../../../assets/bell.svg';
 import Notification from './Notification/Notification';
 import bellLogined from '../../../assets/bellLogined.svg';
-// import useNotification from '../../../hooks/useNotification';
+import useNotification from '../../../hooks/useNotification';
```

ESLint

[error] 'useNotification' is defined but never used.

(@typescript-eslint/no-unused-vars)